src/main/java/stsjorbsmod/cards/cull/ShriekingHat.java

```diff
@@ -0,0 +1,134 @@
+package stsjorbsmod.cards.cull;
+
+import basemod.abstracts.CustomSavable;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.actions.utility.SFXAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardBrieflyEffect;
+import com.megacrit.cardcrawl.vfx.combat.ShockWaveEffect;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.ConsumerGameAction;
+import stsjorbsmod.actions.PermanentlyModifyDamageAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.cards.DeathPreventionCard;
+import stsjorbsmod.cards.WasHPLostCardSubscriber;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.util.CardMetaUtils;
+import stsjorbsmod.util.EffectUtils;
+
+import static stsjorbsmod.JorbsMod.JorbsCardTags.LEGENDARY;
+
+/**
+ * Unplayable. NL Retain. NL When you die instead deal (3 +) all damage taken while this was in your hand to all enemies. NL stsjorbsmod:Destroy.
```

Comment:
I appreciate these comments. I'd like to make this more common.

Reply:
I remember doing it for some cards before, must have started to slip my mind after the break. lol